fluentd/ECS/Json_TaskDefinition

```diff
@@ -0,0 +1,112 @@
+{
+    "ipcMode": null,
+    "executionRoleArn": null,
+    "containerDefinitions": [
+        {
+            "dnsSearchDomains": null,
+            "environmentFiles": null,
+            "logConfiguration": null,
+            "entryPoint": null,
+            "portMappings": [],
+            "command": null,
+            "linuxParameters": null,
+            "cpu": 0,
+            "environment": [
+                {
+                    "name": "APP_NAME",
+                    "value": "ECS"
+                },
+                {
+                    "name": "ENDPOINT",
+                    "value": ""
+                },
+                {
+                    "name": "log_level",
+                    "value": "info"
+                },
+                {
+                    "name": "PRIVATE_KEY",
+                    "value": ""
+                },
+                {
+                    "name": "SUB_SYSTEM",
+                    "value": ""
+                }
+            ],
+            "resourceRequirements": null,
+            "ulimits": null,
+            "dnsServers": null,
+            "mountPoints": [
+                {
+                    "readOnly": null,
+                    "containerPath": "/var/lib/docker",
+                    "sourceVolume": "varlibdocker"
+                },
+                {
+                    "readOnly": true,
+                    "containerPath": "/var/run/docker.sock",
+                    "sourceVolume": "varrundocker"
+                }
+            ],
+            "workingDirectory": null,
+            "secrets": null,
+            "dockerSecurityOptions": null,
+            "memory": null,
+            "memoryReservation": 512,
+            "volumesFrom": [],
+            "stopTimeout": null,
+            "image": "docker.io/natan757/ecs_test:1.0.31",
```

Comment:
fix image name and version

Reply:
changed to: coralogixrepo/fluentd-coralogix-ecs:latest